### ISSUE TITLE
Package `loading size` shoot up 💥💥 Scipy `10 MB -> 39 MB`

### ISSUE THREAD

**Issue by kishorekaruppusamy** [pyodide/pyodide#4175]
## 🐛 Bug

Package loading `size is increased` while loading package from `custom url` using `indexURL` option in loadPyodide API

### To Reproduce

Just download the necessary packages and store it in the server which is mentioned in `indexURL` and run the below html script in browser.

```
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Document</title>
    <script src="https://cdn.jsdelivr.net/pyodide/v0.23.4/full/pyodide.js"></script>
</head>
<body>
    <script>

        let pyodide;

        async function load()
        {
            pyodide = await loadPyodide
            (
                {
                    /**
                    * By Setting this loadPyodide load scripts from the provided server
                    */
                    indexURL : "http://127.0.0.1:55933/"
                }
            );

            await pyodide.loadPackage(["scipy"])
        }

        load()

    </script>
</body>
</html>
```
### Expected behavior
load packages with the size of loading from native cdn server `https://cdn.jsdelivr.net/pyodide/v0.23.4/full/`

### Environment

- Pyodide Version 0.23.4
- Browser version Chrome 116.0.5845.187

Packages loaded from `https://cdn.jsdelivr.net/pyodide/v0.23.4/full/`

<img width="1301" alt="Screenshot 2023-09-26 at 1 25 51 PM" src="https://github.com/pyodide/pyodide/assets/103121724/876c4f2d-9df2-481d-81bb-e5e49e4ca7ac">

Packages loaded from `local python server http://127.0.0.1:55933/`

<img width="1302" alt="Screenshot 2023-09-26 at 1 24 55 PM" src="https://github.com/pyodide/pyodide/assets/103121724/5ad0b5d1-5e8d-46ae-a310-8ba885699cdb">

**Comment by rth** [comment]
Yes, that's expected. The CDN applies Brotli compression which reduces the file size significantly. On your local server you need to also enable compression to get comparable results. Brotli would probably be slow to compress, but say gzip should be fast enough although with a worse compression ratio.

It's a bit confusing in the crime dev console. In Firefox there is more clear separation between actual file size and transferred size accounting for compression.

**Comment by kishorekaruppusamy** [comment]
Thanks for you answer @rth

**Comment by rth** [comment]
Closing as answered, but don't hesitate to comment more.
